dockers/docker-ptf/Dockerfile.template

```diff
@@ -3,7 +3,7 @@ FROM debian:jessie
 MAINTAINER Pavel Shirshov
 
 ## Copy dependencies
-COPY deps /root/deps
+COPY deps /deps
```

Comment:
All docker images need to have debian packages under /deps directory, because generated dockerfile will search for them in /deps directory